_dashboards/im-dashboards/rollover.md

```diff
@@ -1,54 +1,54 @@
 ---
 layout: default
-title: Rollover
+title: Roll over
 parent: Index management in Dashboards
 nav_order: 40
 redirect_from:
   - /dashboards/admin-ui-index/rollover/
 ---
 
-# Rollover
+# Roll over
 Introduced 2.6
 {: .label .label-purple }
 
-OpenSearch Dashboards allows you to perform an [index rollover]({{site.url}}{{site.baseurl}}/im-plugin/ism/error-prevention/index/#rollover) operation with **Index Management**.
+OpenSearch Dashboards allows you to perform an [index roll over]({{site.url}}{{site.baseurl}}/im-plugin/ism/error-prevention/index/#rollover) operation with **Index Management**.
```

Comment:
Should we explain why someone would want to do this?

Reply:
"with Index Management"? How about, "You can perform a roll over operation in OpenSearch Dashboards when you want to ...."

Reply:
This is explained in the ISM/data-stream/API documentation as to the “why” you would do it. As this is a front-end for an existing feature I wouldn’t think so. Also, I have linked to that content in this page.